changes/ee/feat-15064.en.md

```diff
@@ -0,0 +1,27 @@
+New Experimental cluster Linear Session Registry (LSR)
+
+Added support for handling large-scale client reconnections. 
+This feature prioritizes accepting the most recent client connection attempt over rejecting it in favor of older connections.
+
+Enable the feature by configuring:
+
+```
+broker.enable_linear_session_registry = true
+broker.enable_session_registry = false
```

Comment:
this flag was to turn off global session registration completely.
i.e. no cross-node session take-over at all.
I think we should keep this flag as it is intended:
`true`: either LSR or the legacy registration
`false`: no registration at all, no LSR, no legacy registration.

Reply:
Yes. it is indeed to turn off global session registration completely becase LSR is a replacement of `global session registration`,  node could read from LSR to complte cross node session takeover/discard.

EIP described the need of parallel enabling of both LSR and legacy `global` session registry to possibley handle rolling upgrades. (not addressed in this PR yet)

Reply:
My bad I didn't write the doc for this experimental feature, here I updated it. 
https://github.com/emqx/emqx/pull/15064/commits/a79f17171461fea318144e92f514a2bda886f6fa

LSR provides global registry but it doesn't want to be controlled by `enable_session_registry` becasue it is used by `emqx_cm_registry` which is the old 'session registry' and has other bugs that I don't want to touch in this PR.

having two options to control both has the flexibility to operator like for upgrade or feature enabling.